README.md

```diff
@@ -124,16 +124,51 @@ Use [Homebrew] to install the packages:
 brew install sdl2 ffmpeg
 
 # client build dependencies
-brew install gcc pkg-config meson
+brew install pkg-config meson
 ```
 
-Java (>= 7) is not available in Homebrew, so if you plan to build the server,
-install it manually and make it available from the `PATH`:
-
+Additionally, if you want to build the server, install Java 8
+from Caskroom, and make it avaliable from the `PATH`:
 ```bash
+brew tap caskroom/versions
+brew cask install java8
+export JAVA_HOME="$(/usr/libexec/java_home --version 1.8)"
 export PATH="$JAVA_HOME/bin:$PATH"
 ```
 
+If you don't need to build the server, you can use [this unofficial Homebrew formula](https://github.com/stek29/homebrew-core/blob/3eb5ea5083fc8285d00a4363d64ea3bf68bc98dd/Formula/scrcpy.rb)
+to install scrcpy.
```

Comment:
I saw you did a [PR on homebrew](https://github.com/Homebrew/homebrew-core/pull/25173), this is great! Thank you. :+1:

I am not very able to review this (for example the fakeadb part), so I'd prefer to remove this paragraph and wait for it to be available (or at least reviewed on your PR).

Reply:
> Actually, second commit might be not really fitting, especially Homebrew part.

Let's wait for PR to be merged :)

Reply:
> Actually, second commit might be not really fitting, especially Homebrew part.

What do you want to change? Should I merge only the first commit now?

Reply:
@rom1v I want it to be merged into homebrew-core so user can do `brew install scrcpy && brew cask install android-platform-tools` to get everything working